src/renderer/component/wunderbar/view.jsx

```diff
@@ -16,15 +19,23 @@ type Props = {
   doFocus: () => void,
   doBlur: () => void,
   resultCount: number,
+  focused: boolean,
 };
 
 class WunderBar extends React.PureComponent<Props> {
-  constructor(props: Props) {
-    super(props);
+  constructor() {
+    super();
 
     (this: any).handleSubmit = this.handleSubmit.bind(this);
     (this: any).handleChange = this.handleChange.bind(this);
-    this.input = undefined;
+  }
+
+  componentDidMount() {
+    window.addEventListener('keydown', this.handleKeyDown.bind(this));
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('keydown', this.handleKeyDown);
```

Comment:
Does this correctly remove the event?  Using `bind` above creates a handle that isn't stored.

Reply:
👍 I'll move the `bind` into the constructor and use `this.handleKeyDown` for both.